modules/ROOT/pages/install-self-managed.adoc

```diff
@@ -71,6 +102,37 @@ curl -L https://devx.anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/lat
 sudo chmod +x rtfctl
 ----
 
+== Create a Runtime Fabric using Runtime Manager
+
```

Comment:
We have at least two different personas performing actions on this doc: the *IT administrator* who is validating network requirements and installing RTF, and the *MuleSoft organizational administrator* who performs the actions within Anypoint Platform.

With that in mind, could we add a statement under this section to indicate these steps should be provided by a MuleSoft organization administrator?

Reply:
Added a note at the beginning of the topic:

- Runtime Fabric installation includes tasks that must be peformed by both IT administrators and MuleSoft organization administrators.

Also added a note at the beginning of each section:

- The procedures in this section should be performed by a MuleSoft organization administrator.